### DISCUSSION TITLE
How to get an github achievements?

### DISCUSSION THREAD

**Discussion by sneakybeakyq** [community/community#55447] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I dont even know

**Comment by emilycodestar** [comment]
Here is instruction: https://github.com/4xmen/Get-Github-Achievements/blob/main/assets/steps/README.publicsponsor.md

**Comment by queenofcorgis** [comment]
@sneakybeakyq @dinosaurtirex GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answering and locking the post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
